src/test/java/org/junit/platform/engine/test/TestDescriptorStub.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.platform.engine.test;
+
+import org.junit.platform.engine.UniqueId;
+import org.junit.platform.engine.support.descriptor.AbstractTestDescriptor;
+
+/**
+ * @since 1.0
+ */
+public class TestDescriptorStub extends AbstractTestDescriptor {
+
+	public TestDescriptorStub(UniqueId uniqueId, String displayName) {
```

Comment:
Does the super constructor already check the nullness of these parameters?

If not, have you considered using `Objects.requireNonNull` or Guava's `Preconditions.checkNotNull`?